Document independent full and differential backup scheduling (#2998)
neo4j/docs-operations: modules/ROOT/pages/backup-restore/online-backup.adoc

A _backup chain_ consists of a full backup optionally followed by a sequence of n contiguous differential backups.

[NOTE]
====

> Starting in 2026.02, the first differential backup in a chain may _overlap_ with the full backup -- that is, its lowest transaction ID may be less than or equal to the full backup's highest transaction ID.

Reviewer: ```suggestion
Starting with 2026.02, the first differential backup in a chain may _overlap_ with the previous full backup, meaning its lowest transaction ID may be less than or equal to the full backup's highest transaction ID.
```

Note that this will speed up the backup process if sufficient throughput is provisioned for both the backup process and the source servers from which data is being pulled.

[role=label--new-2026.02]
[[backup-scheduling-rpo-rto]]
=== Scheduling for RPO and RTO

> Starting in 2026.02, the first differential backup in a xref:backup-restore/online-backup.adoc#backup-chain[backup chain] may overlap with its parent full backup.

Reviewer: ```suggestion
Starting with 2026.02, the first differential backup in a xref:backup-restore/online-backup.adoc#backup-chain[backup chain] may overlap with its parent full backup.
```

This allows you to schedule full and differential backups independently, tuning each cadence to a different recovery objective:

Recovery Point Objective (RPO)::
The maximum amount of data loss you can tolerate, measured in time.

> Drive RPO with the *differential* backup schedule -- a diff every 15 minutes yields an RPO of roughly 15 minutes.

Reviewer: ```suggestion
Drive RPO with the *differential* backup schedule -- a differential backup every 15 minutes yields an RPO of roughly 15 minutes.
```

Recovery Time Objective (RTO)::
The maximum time a restore is allowed to take.

> Drive RTO with the *full* backup schedule -- a more recent full means a shorter diff chain to replay at restore time, which means a faster restore.

Reviewer: ```suggestion
Drive RTO with the *full* backup schedule -- a more recent full backup shortens a differential chain that must be replayed at restore time, enabling faster recovery.
```

> Before 2026.02, a full backup taken between two scheduled differential backups would break the diff chain, because the next diff had to start exactly where the new full ended.

Reviewer: ```suggestion
Before 2026.02, a full backup taken between two scheduled differential backups would break the differential chain, because the next differential backup had to start exactly where the new full backup ended.
```

> In practice this forced the two schedules to be coupled, and a full backup would interrupt the diff cadence, degrading RPO.

Reviewer: ```suggestion
In practice this forced the two schedules to be coupled, and a full backup would interrupt the differential cadence, degrading RPO.
```

With overlap allowed, two independent processes can run safely:

* A differential schedule set to the desired RPO (for example, every 15 minutes).
* A full schedule set to the desired RTO (for example, daily or weekly).

> The first differential backup after each full will naturally overlap the full; the chain remains restorable.

Reviewer: ```suggestion
The first differential backup after each full backup naturally overlaps with it; the chain remains restorable.
```

[NOTE]
====
A differential backup can only succeed if there is already a xref:backup-restore/online-backup.adoc#backup-chain[backup chain] to build upon -- that is, at least one full backup must be present in the target location.

You have two options when setting up the schedules:

* Seed the target location with an initial full backup (or ensure the full schedule has produced one) before the differential schedule starts running.
* Run the differential schedule with `--type=AUTO` (the default) instead of `--type=DIFF`.

> With `AUTO`, the backup client falls back to a full backup when no chain is present, so the first run will produce a full and subsequent runs will produce diffs.

Reviewer: ```suggestion
With `AUTO`, the backup client falls back to a full backup when no chain is present.
Therefore, the first run produces a full backup and subsequent runs produce differential backups.
```
